docs/operator-manual/monitor-mode.md

```diff
@@ -57,39 +50,25 @@ spec:
     - NET_ADMIN
 ```
 
-## Transitioning policy mode
-
-It's worth noting that certain attributes can be updated on policies
-once they have been deployed.
-
-For security purposes, a user with UPDATE permissions on policy
-resources can make the policy more restrictive, so that:
+## Changing policy mode
 
-- Transitioning the `mode` of an existing `ClusterAdmissionPolicy` or
-  `AdmissionPolicy` from `monitor` to `protect` is allowed.
+It's worth noting that you can update certain attributes on policies
+that are already deployed.
 
-However,
+For security purposes, a user with `UPDATE` permissions on policy resources can make the policy more restrictive.
+This means that you can change the `mode` of an existing `ClusterAdmissionPolicy` or `AdmissionPolicy` from `monitor` to `protect`.
 
-- Transitioning the `mode` of an existing `ClusterAdmissionPolicy` or
-  `AdmissionPolicy` from `protect` to `monitor` is **disallowed**.
-This is because the policy would be effectively disabled during
-  the time that it would have been in `monitor` mode, until it was
-  restored back to `protect` mode.
+However, you can't change the `mode` of an existing `ClusterAdmissionPolicy` or `AdmissionPolicy` from `protect` to `monitor`.
+Changing to `monitor` mode would 'switch off' the policy and so a security issue, until restored to `protect` mode.
```

Comment:
I think this phrase could be dropped, as I don't really understand it. I think it is fine stating that we only allow for policy changes into more restrictive mode as in line 58.

Reply:
Ok, I think I have that. See what you think in the revised text.